tests/unit/model_test.py

```diff
@@ -83,4 +84,14 @@ def test_run_async_errors(status_code, error_message):
         test_model = Model(id=model_id, name="Test Model",url=base_url)
         response = test_model.run_async(data="input_data")
     assert response["status"] == "FAILED"
-    assert response["error_message"] == error_message
\ No newline at end of file
+    assert response["error_message"] == error_message
+
+def test_model_io():
+    model_id = "64aee5824d34b1221e70ac07"
+    model = ModelFactory.get(model_id)
+
+    expected_input = {"text"}
```

Comment:
Is this passing? A dictionary usually requires a value:

```
{ "text': "..." }
```
Please also check the other model tests are passing after the changes.